tensorbay/opendataset/CACD/loader.py

```diff
@@ -0,0 +1,104 @@
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+# pylint: disable=missing-module-docstring
+# pylint: disable=line-too-long
+
+import os
+from typing import Any, Dict, Tuple
+
+import numpy as np
+from h5py import File
```

Comment:
`h5py` is not a SDK requirement, make it a optional requirement by importing it in function.